monitoring/prometheus-config.yml

```diff
@@ -213,4 +213,84 @@ scrape_configs:
       - waku-simulator_nwaku_198:8008
       - waku-simulator_nwaku_199:8008
-      - waku-simulator_nwaku_200:8008
\ No newline at end of file
+      - waku-simulator_nwaku_200:8008
+      - liteprotocoltester-filter-service-1:8008
+      - liteprotocoltester-filter-service-2:8008
+      - liteprotocoltester-filter-service-3:8008
+      - liteprotocoltester-filter-service-4:8008
+      - liteprotocoltester-filter-service-5:8008
+      - liteprotocoltester-filter-service-6:8008
+      - liteprotocoltester-filter-service-7:8008
+      - liteprotocoltester-filter-service-8:8008
+      - liteprotocoltester-filter-service-9:8008
+      - liteprotocoltester-filter-service-10:8008
+      - liteprotocoltester-filter-service-11:8008
+      - liteprotocoltester-filter-service-12:8008
+      - liteprotocoltester-filter-service-13:8008
+      - liteprotocoltester-filter-service-14:8008
+      - liteprotocoltester-filter-service-15:8008
+      - liteprotocoltester-filter-service-16:8008
+      - liteprotocoltester-filter-service-17:8008
+      - liteprotocoltester-filter-service-18:8008
+      - liteprotocoltester-filter-service-19:8008
+      - liteprotocoltester-filter-service-20:8008
+      - liteprotocoltester-lightpush-service-1:8008
+      - liteprotocoltester-lightpush-service-2:8008
+      - liteprotocoltester-lightpush-service-3:8008
+      - liteprotocoltester-lightpush-service-4:8008
+      - liteprotocoltester-lightpush-service-5:8008
+      - liteprotocoltester-lightpush-service-6:8008
+      - liteprotocoltester-lightpush-service-7:8008
+      - liteprotocoltester-lightpush-service-8:8008
+      - liteprotocoltester-lightpush-service-9:8008
+      - liteprotocoltester-lightpush-service-10:8008
+      - liteprotocoltester-lightpush-service-11:8008
+      - liteprotocoltester-lightpush-service-12:8008
+      - liteprotocoltester-lightpush-service-13:8008
+      - liteprotocoltester-lightpush-service-14:8008
+      - liteprotocoltester-lightpush-service-15:8008
+      - liteprotocoltester-lightpush-service-16:8008
+      - liteprotocoltester-lightpush-service-17:8008
+      - liteprotocoltester-lightpush-service-18:8008
+      - liteprotocoltester-lightpush-service-19:8008
+      - liteprotocoltester-lightpush-service-20:8008
+      - liteprotocoltester-receivernode-1:8008
+      - liteprotocoltester-receivernode-2:8008
+      - liteprotocoltester-receivernode-3:8008
+      - liteprotocoltester-receivernode-4:8008
+      - liteprotocoltester-receivernode-5:8008
+      - liteprotocoltester-receivernode-6:8008
+      - liteprotocoltester-receivernode-7:8008
+      - liteprotocoltester-receivernode-8:8008
+      - liteprotocoltester-receivernode-9:8008
+      - liteprotocoltester-receivernode-10:8008
+      - liteprotocoltester-receivernode-11:8008
+      - liteprotocoltester-receivernode-12:8008
+      - liteprotocoltester-receivernode-13:8008
+      - liteprotocoltester-receivernode-14:8008
+      - liteprotocoltester-receivernode-15:8008
```

Comment:
Mind giving some context on what each type of node does?
how is publishernode different from lightpush-service? and filter vs receivednode?

Reply:
Yes, this one I could not separate hence including foreign services here.
Mind that my setup is running in a separate compose which is interconnected with simulator via the docker network it defines.
But to see my services on the board I needed to add them here.
In my setup I have n-number of lightpush client connects to a lightpush-service node and a filter client that is connected with a separate filter-service node... where service nodes are connected to waku-simulator relay network.

Reply:
See here: https://github.com/waku-org/nwaku/pull/2813